<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWidget</class>
 <widget class="QMainWindow" name="GraphWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gvisual</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Граф</string>
    </property>
    <addaction name="a_create"/>
    <addaction name="a_open"/>
    <addaction name="a_save"/>
    <addaction name="a_save_as"/>
    <addaction name="separator"/>
    <addaction name="a_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="a_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="a_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="a_create">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="a_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="a_save_as">
   <property name="text">
    <string>Сохранить как..</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
